deferral/ah-django-frontend
# 164859698 layout the boilerplate

### What does this PR do?

Set up the project boilerplate

### Description of Task to be completed?

[x] Use create-react-app to bootstrap the layout
[x] Install the Airbnb as the default linting tool

### How should this be manually tested?

Access the repository on GitHub from [here](https://github.com/deferral/ah-django-frontend.git)
Switch to the branch ch-setup-boilerplate-164859698
You should see a directory called app with all the setup files

### Any background context you want to provide?

The create-react-app has the essentials that sets up the basis of developing the frontend app

### What are the relevant pivotal tracker stories?

[#164859698](https://www.pivotaltracker.com/story/show/164859698)
